test/e2e-cypress/tests/features/oauth2-flows/password.js

```diff
@@ -30,7 +30,7 @@ describe("OAuth2 Password flow", function() {
       .clear()
       .type("secret")
 
-      .get("div.modal-ux-content > div:nth-child(1) > div > div:nth-child(2) > div > div.auth-btn-wrapper > button.sui-btn.modal-btn.authorize")
+      .get("div.modal-ux-content > div:nth-child(1) > div > div:nth-child(2) > div > div.auth-btn-wrapper > button.sui-btn--primary")
```

Comment:
Yup. Didn't think of that :D 
I take it you'd like to avoid the use of IDs as much as possible?

Reply:
yeah, a couple of reasons:
* you can only have one element with an ID in your DOM... adopting non-dynamic IDs could cause problems for folks that have multiple Swagger UI instances on their page at once
* having IDs around encourages folks to extend Swagger UI by using native listeners and such that target the native element by ID, rather than using our plugin system